src/components/ModalExport.tsx

```diff
@@ -113,6 +113,13 @@ export default class ModalExport extends React.Component<ModalExportProps> {
         <p>
           Download a JSON style to your computer.
         </p>
+        <p>
+          <a href="https://docs.maptiler.com/cloud/api/authentication-key/" target="_blank" rel="noreferrer">MapTiler</a>,&nbsp;
+          <a href="https://www.thunderforest.com/docs/apikeys/" target="_blank" rel="noreferrer">ThunderForest</a>,
+          and <a href="https://docs.stadiamaps.com/authentication/" target="_blank" rel="noreferrer">Stadia Maps</a>&nbpsp;
```

Comment:
Trailing `&nbsp;` is because otherwise the following text becomes illegibly squashed.